File: java/src/main/java/mediator/message/Message.java
package mediator.message;

public class Message {
    private String service;
    private String type;

    public Message(String service, String type) {
        this.service = service;
        this.type = type;
    }

    public String getService() {
        return service;
    }

    public void setService(String service) {
        this.service = service;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: csharp/SEP3WebAPI/Mediator/Messages/Message.cs
namespace SEP3WebAPI.Mediator.Messages {
    public class Message {
        public string Service { get; set; }
        public string Type { get; set; }

        /**
         * The service is set automatically when instantiating the specific classes
         */
         public Message(string service) {
            Service = service;
        }
    }
}